packages/currency/src/erc20/index.ts

```diff
@@ -17,6 +17,7 @@ export function getSupportedERC20Tokens(): ERC20Currency[] {
           network: networkName,
           decimals: token.decimals,
           symbol: token.symbol,
+          id: token.id || `${token.symbol}-${networkName}`,
```

Comment:
This is redundant, there should be only 1 place where we compute the ID
```suggestion
          id: token.id,
```

Reply:
Sorry, this was my suggestion. @KolevDarko I guess its not needed because the ID is already added here, in `fromInput()`: https://github.com/RequestNetwork/requestNetwork/blob/master/packages/currency/src/currencyManager.ts#L311